devices.ttl

```diff
@@ -1,5 +1,13 @@
 @prefix device: <http://example.org/vocab/device/> .
 
+_:label device:make "Teenage Engineering" .
+_:label device:model "EP133 K.O. II" .
+_:label device:inputs 1 .
+_:label device:outputs 1 .
+_:label device:swappable true .
```

Comment:
`swappable` means the device is can be configured for both Type A and Type B. Is that true for the EP–133 K.O.II? I wasn’t able to find any docs that indicate it can support Type B.